packages/react-data-grid-examples/package.json

```diff
@@ -25,9 +25,7 @@
   "devDependencies": {
     "bootstrap": "^3.2.0",
     "jquery": "^2.1.1",
-    "lodash": "^4.13.1",
-    "lodash.groupby": "^4.5.1",
     "markdown": "^0.5.0",
-    "tape": "^4.9.1"
+    "recharts": "^1.5.0"
```

Comment:
I wonder if we shouldn't just move all the deps under devDependencies for the examples.

Reply:
I think we should just delete the examples project and migrate all the examples to the new website

Reply:
Let's do that yeah.